### PR TITLE
feat(sixtyfps): add support for SixtyFPS ls

### PR DESCRIPTION
**Pull request by RustemB**

### CODE REVIEW

File: lua/lspconfig/sixtyfps.lua [hunk 1]
```diff
@@ -0,0 +1,21 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.sixtyfps = {
+    default_config = {
+        cmd = {'sixtyfps-lsp'},
+        filetypes = {'sixtyfps'},
```
Comment (mjlbach) [inline lua/lspconfig/sixtyfps.lua:7]: Is there a filetype plugin users can install so this is recognized?

File: lua/lspconfig/sixtyfps.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        root_dir = function(filename) return util.path.dirname(filename) end
```
Comment (mjlbach) [inline lua/lspconfig/sixtyfps.lua:8]: There has to be a better root marker than this

Comment (RustemB) [reply lua/lspconfig/sixtyfps.lua:8]: I don't think that it has it
It's just a markup language, that can be used in Rust, C++ and NodeJS projects

File: lua/lspconfig/sixtyfps.lua (hunk 1 above)
Comment (RustemB) [reply lua/lspconfig/sixtyfps.lua:7]: Not yet, but can be added manually at this moment, 'till language development in active progress

File: lua/lspconfig/sixtyfps.lua (hunk 2 above)
Comment (mjlbach) [reply lua/lspconfig/sixtyfps.lua:8]: I'm just generally opposed to adding roots that could be invalid, I may change the mechanism such that servers launch without project roots after 0.5.1 is released.

File: lua/lspconfig/sixtyfps.lua (hunk 1 above)
Comment (RustemB) [reply lua/lspconfig/sixtyfps.lua:7]: https://github.com/RustemB/sixtyfps-vim

Comment (mjlbach) [reply lua/lspconfig/sixtyfps.lua:7]: Can you mention that you need this filetype plugin in the docs?

Comment (RustemB) [reply lua/lspconfig/sixtyfps.lua:7]: ok

File: lua/lspconfig/sixtyfps.lua [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
```
Comment (lithammer) [inline lua/lspconfig/sixtyfps.lua:8]: I'd say replace this with `util.find_git_ancestor` and then add `single_file_support = true`. On the other hand, I'm not sure how SixtyFPS handles a `null` root.

 ```suggestion
        root_dir = util.find_git_ancestor,
        single_file_support = true,
```

Comment (mjlbach) [reply lua/lspconfig/sixtyfps.lua:8]: That was my concern too :/ I need to test

**Comment by mjlbach** [comment]
I'm closing this PR in favor of #1430 for the sole reason that since you used master instead of a feature branch I am now locked in a battle with our autogenerated doc

**Pull request closed without merging**